Dockerfile.dist

```diff
@@ -17,9 +17,11 @@
 ## Build
 
 # This works fine but will produce an EC violation
-# FROM docker.io/library/golang:1.21 AS build
+# FROM docker.io/library/golang:1.22 AS build
```

Comment:
Do we need these comments, perhaps its a good opportunity to remove them?

Reply:
Good idea.